projects/core/src/lib/plots/plot-list-entry/plot-list-entry.component.scss

```diff
@@ -3,7 +3,7 @@ img {
     height: auto;
 }
 
```

Comment:
icons below plots are very small and not centered

![image](https://user-images.githubusercontent.com/1027102/210561472-1cf847ba-e3b5-4076-b3f0-2c4e9a09fce9.png)

Reply:
They are now default icon buttons:
[8d44fa4](https://github.com/geo-engine/geoengine-ui/pull/555/commits/8d44fa47bbc0e4a1d1464b63ebd6052394ac0c8a)